core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -78,7 +78,21 @@ static PageStore create(AlluxioConfiguration conf) {
    * @throws IOException
    * @throws PageNotFoundException when the page isn't found in the store
    */
-  ReadableByteChannel get(PageId pageId) throws IOException,
+  default ReadableByteChannel get(PageId pageId) throws IOException,
+      PageNotFoundException {
+    return get(pageId, 0);
+  }
+
+  /**
+   * Gets part of a page from the store to the destination channel.
+   *
+   * @param pageId page identifier
+   * @param pageOffset offset within page
+   * @return the number of bytes read
+   * @throws IOException
+   * @throws PageNotFoundException when the page isn't found in the store
+   */
+  ReadableByteChannel get(PageId pageId, int pageOffset) throws IOException,
```

Comment:
what is expected if pageOffset > page len? Exception or an empty channel?

Reply:
Will throw an `IllegalArgumentException` just to be safe.